srcpkgs/rpm/template

```diff
@@ -1,22 +1,22 @@
 # Template file for 'rpm'
 pkgname=rpm
-version=4.15.1
-revision=4
+version=4.16.0
+revision=1
 build_style=gnu-configure
 configure_args="--with-lua --with-cap --with-acl --with-external-db
  --enable-python PYTHON=python3 --sharedstatedir=/var/lib"
 hostmakedepends="automake gettext-devel libtool nss-devel pkg-config
  python3-setuptools"
-makedepends="binutils-devel db-devel elfutils-devel file-devel libarchive-devel
+makedepends="libgcrypt-devel binutils-devel db-devel elfutils-devel file-devel libarchive-devel
  libcap-devel lua52-devel nss-devel popt-devel python3-devel libgomp-devel"
 short_desc="Red Hat Package Management Utils"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later"
 homepage="https://rpm.org"
 changelog="https://rpm.org/wiki/Releases/${version}.html"
-distfiles="http://ftp.rpm.org/releases/rpm-4.15.x/rpm-${version}.tar.bz2"
-checksum=ddef45f9601cd12042edfc9b6e37efcca32814e1e0f4bb8682d08144a3e2d230
-python_version=2 #unverified
+distfiles="http://ftp.rpm.org/releases/rpm-${version%.*}.x/rpm-${version}.tar.bz2"
+checksum=ca5974e9da2939afb422598818ef187385061889ba766166c4a3829c5ef8d411
+python_version=2
```

Comment:
Is 2 correct? It's linked against python3...

Reply:
deleting this line, the package does not compile.

Reply:
No, I think you should be using `python_version=3`

Reply:
ping here

Reply:
It seems that there is exactly one Python script installed by `rpm`, and it works with Python 3 (I can at least get the help text to display). Please change to `python_version=3` as @ericonr recommends.